tests/os/freebsd/snmp/list-diskspath.robot

```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Documentation       Check list-diskspath table
+
+Resource            ${CURDIR}${/}..${/}..${/}..${/}resources/import.resource
+
+Test Timeout        120s
+
+
+*** Variables ***
+${CMD}      ${CENTREON_PLUGINS} --plugin=os::freebsd::snmp::plugin
+
+
+*** Test Cases ***
+list-diskspath ${tc}
+    [Tags]    os    freebsd
+    ${command}    Catenate
+    ...    ${CMD}
+    ...    --mode=list-diskspath
+    ...    --hostname=${HOSTNAME}
+    ...    --snmp-version=${SNMPVERSION}
+    ...    --snmp-port=${SNMPPORT}
+    ...    --snmp-community=os/linux/snmp/linux
+    ...    --snmp-timeout=1
+    ...    ${extra_options}
+
+    Ctn Run Command And Check Result As Strings    ${command}    ${expected_result}
+
+    Examples:        tc    extra_options                           expected_result    --
+            ...      1     --skip-total-size-zero=''               List disk path: ${SPACE}'/run/user/0' [id = 25] ${SPACE} '/run' [id = 5] ${SPACE} '/' [id = 6] ${SPACE} '/dev/shm' [id = 8] ${SPACE} '/run/lock' [id = 9]
```

Comment:
I tryied whithout the ${space} variable this file still work, why did you add it here ?